training-net/src/Controllers/MoviesController.cs

```diff
@@ -1,19 +1,25 @@
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
 using System.Text.Encodings.Web;
 
 namespace MvcMovie.Controllers
 {
     public class MoviesController : Controller
     {
-        public IActionResult Index()
+        private readonly IHtmlLocalizer<MoviesController> _localizer;
+        
+        public MoviesController(IHtmlLocalizer<MoviesController> localizer)
         {
+            this._localizer = localizer;
+        }
+        public IActionResult Index()
+        {   
+            ViewData["Message"] = _localizer["ContactPage"];
```

Comment:
we only user de readonly attribute (`_localizer`) on the constructor. after, if we need to use it, we declare a property and use it. here, for example:
`public IHtmlLocalizer<MoviesController> Localizer
{
    get { return this._localizer; }
}`

Reply:
also.. where are you showing this ViewData["Message"]? i can't see anywhere if the localizer is working or not